ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/PsiUtil.fs

```diff
@@ -444,6 +444,11 @@ let rec tryFindRootPrefixAppWhereExpressionIsFunc ([<CanBeNull>] expr: IFSharpEx
         tryFindRootPrefixAppWhereExpressionIsFunc(prefixApp)
     else expr
 
+let rec getRootFunctionExpr ([<NotNull>] app : IPrefixAppExpr) =
```

Comment:
Any reason to have it in a common util? Are there other uses that could be replaced?

Reply:
OK, now, when it was moved inside the quick fix, are you sure there are no existing uses that could benefit from having it in the util?

Reply:
Now I noticed that `IPrefixAppExpr` already has the property `InvokedReferenceExpression`, necessary for this (: